platform/ios/app/MBXOrnamentsViewController.m

```diff
@@ -64,7 +64,7 @@ - (void)viewDidLoad {
                        zoomLevel:16
                        direction:30
                         animated:NO];
-    mapView.delegate = self;
+    
     mapView.showsScale = YES;
```

Comment:
Why are we removing this?

Reply:
This was dead code in the examples that surfaced with the introduction of the new delegate method.